### ISSUE TITLE
match details

### ISSUE THREAD

**Issue by tyrcho** [elovation/elovation#54]
Excellent ! I've hosted it to Heroku in 10 min without knowing anything about Rails !

I plan to use this for a group of Hearthstone players.

I'd like to enter some details about the match (an URL, a score, a date).

Also when creating a match it would be nice that I can't face myself in the autocomplete fields.

**Comment by Hawke** [comment]
I agree — being able to backdate scores would be fantastic (Of course it would require recalculating a large subset of ratings of everything newer)
